### PR TITLE
TaskletFusion and Mish example

### PR DESCRIPTION
**Pull request by orausch**

### CODE REVIEW

File: examples/plot_cuda_mish.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
+"""
+Optimizing the Mish Operator
+============================
+
+DaCeML allows users to optimize DNN modules at all levels of granularity, from operators to full models. In this
+example, we optimize the Mish operator [1]_, a relatively novel activation function that,
+among other uses, has been applied successfully in image segmentation. [2]_
```
Comment (tbennun) [inline examples/plot_cuda_mish.py:7]: object detection

File: examples/plot_cuda_mish.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+# %%
+# We also see another opportunity for optimization: The DaCeML autodiff engine is "forwarding" intermediate values to
+# perform the differentiation. This means that the intermediate values have to be written out in the forward pass, and
+# and read in the backward pass.
+#
+# Optimization
+# ------------
+#
+# To improve the runtime, we'll apply 3 transformations.
```
Comment (tbennun) [inline examples/plot_cuda_mish.py:129]:
```suggestion
# To improve the runtime, we'll apply three transformations.
```

File: examples/plot_cuda_mish.py [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+#
+# Firstly, we'll use ``SubgraphFusion`` to fuse all the maps into a single kernel.
+# To tackle the second issue, we'll use the :class:`~daceml.transformation.TaskletFusion` transformation. By fusing the
```
Comment (tbennun) [inline examples/plot_cuda_mish.py:132]:
```suggestion
# To tackle the issue of forwarding intermediate values in backprop, we'll use the :class:`~daceml.transformation.TaskletFusion` transformation. By fusing the
```

**Review by tbennun — CHANGES_REQUESTED** [review]
very minor changes, otherwise approved :)

File: examples/plot_cuda_mish.py (hunk 1 above)
Comment (orausch) [reply examples/plot_cuda_mish.py:7]:
```suggestion
among other uses, has been applied successfully in object detection. [2]_
```

**Review by tbennun — APPROVED** [review]

**Pull request merged**